apps/cnquery/cmd/builder/common/common.go

```diff
@@ -508,6 +508,27 @@ func ScanGcpComputeInstanceCmd(commonCmdFlags CommonFlagsFn, preRun CommonPreRun
 	return cmd
 }
 
+func ScanGcpComputeSnapshotCmd(commonCmdFlags CommonFlagsFn, preRun CommonPreRunFn, runFn RunFn, docs CommandsDocs) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "snapshot INSTANCE-NAME",
+		Short: docs.GetShort("gcp-compute-instance"),
+		Long:  docs.GetLong("gcp-compute-instance"),
```

Comment:
The docs are missing. And should these be the same as for the `scan gcp instance` command?

Reply:
```
Available Commands:
  folder      Scan a Google Cloud Platform (GCP) folder.
  gcr         Scan a Google Container Registry (GCR).
  instance
  org         Scan a Google Cloud Platform (GCP) organization.
  project     Scan a Google Cloud Platform (GCP) project.
  snapshot
  ```

Reply:
added the content to the shell and the scan subcommand